### PR TITLE
Remove HWID from default key to add VM instance to DUT list

### PR DESCRIPTION
**Pull request by 4n12i**
Remove HWID from the list of the default DUT info key to add VM instances (e.g. betty VM) to the DUT list

### CODE REVIEW

**Comment by 4n12i** [comment]
In this issue https://github.com/google/lium/issues/204, we considered using another key.   
However, since I found that the betty VM has the model name and MAC address, it was able to obtain the DUT information by simply removing the HWID from the list of the default DUT info keys.

File: src/dut.rs [hunk 1]
```diff
@@ -180,10 +181,10 @@ const CMD_GET_DEFAULT_IFACE: &str =
     r"ip route get 8.8.8.8 | sed -E 's/^.* dev ([^ ]+) .*$/\1/' | head -n 1";
 
 // Only keys that are always available can be listed here
-const DEFAULT_DUT_INFO_KEYS: [&str; 8] = [
+// We should consider the value of these keys if DUT is a VM
```
Comment (mhiramat) [inline src/dut.rs:184]: This comment is not needed because this is not only for the VM.

File: src/dut.rs [hunk 2]
```diff
@@ -143,6 +143,7 @@ lazy_static! {
     static ref DUT_ATTRIBUTE_CMDS: HashMap<&'static str, &'static str> = {
         let mut m: HashMap<&'static str, &'static str> = HashMap::new();
         m.insert("board", r"cat /etc/lsb-release | grep CHROMEOS_RELEASE_BOARD | cut -d '=' -f 2 | cut -d '-' -f 1");
+        // VM instances do not have a valid HWID.
```
Comment (mhiramat) [inline src/dut.rs:146]: Could you add 'NOTE: '  at the beginning of this comment?

File: src/dut.rs (hunk 1 above)
Comment (4n12i) [reply src/dut.rs]: OK, I removed it!

File: src/dut.rs (hunk 2 above)
Comment (4n12i) [reply src/dut.rs]: Fixed!

**Review by hikalium — APPROVED** [review]
LGTM. Thank you!!

**Pull request merged**